examples/transfer_files/transfer_files.py

```diff
@@ -49,8 +47,8 @@
 from typing_extensions import Annotated
 
 ###################################################
-# Necessary imports
-# =================
+# Perform necessary imports
```

Comment:
Given that this code block has a heading, put a heading before the preceding code block? This would allow the reader to use the right pane to jump to either of the two subheadings.